docs/diagnostics/PX1007.md

```diff
@@ -15,13 +15,28 @@ The code fix does one of the following:
  - Adds the `summary` XML tags, in which you type the description of the code item
  - Adds the `exclude` XML tag, which excludes the item from the API and DAC References
 
-The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is set to _True_. By default, the value of this option is _False_. 
-
-This diagnostic is intended for Acumatica ERP developers, however it can be used by any developers to provide documentation of source code.
-
-The diagnostic is currently displayed for all public entities (classes, interfaces, structs, enumerations, and delegates) and DAC property fields no matter in which namespace they are located. The diagnostic is not displayed for other fields, properties, and methods of public entities. 
-
-The diagnostic is not displayed for child classes if their parent class has the `exclude` XML tag.
+The new diagnostic is available only if the value of **Tools > Options > Acuminator > Code Analysis > Enable PX1007 diagnostic** is _True_. By default, the value of this option is _False_. 
+
+This diagnostic is intended for Acumatica ERP developers; however, it can be used by any developer to provide documentation of the source code.
+
+The diagnostic is currently displayed for the following public items:
+ - Classes that are not DACs 
+ - DACs with the `PXCacheName` attribute
```

Comment:
@k-s-popova this is not correct. The diagnostic is displayed for all public classes without xml comment unless they have one of the attributes: `PXHidden`, `PXInternalUseOnly`, or `Obsolete`. 
In case the class has one of the mentioned attributes the diagnostic won't be shown not only for it but also for all its members suitable for the diagnostic - public nested types, DAC properties.
This attributes work like "exclude" tag.

Reply:
@SENya1990 , it was in the requirements in the JIRA issue to display the diagnostics for the DACs with PXCacheName, cause we are not interested in other DACs for the DAC reference.

Reply:
@k-s-popova but the DAC is still a public class in C# and the diagnostic requires all public classes to be documented.  Why should it be excluded? Moreover, having a DAC without either PXCacheName or PXHidden is also an error - we have another diagnostic for it. 
We added support to not show PX1007 for PXHidden. So, it's imho not good to check additionally for PXCacheName. It will be more costly without adding real value

Reply:
@SENya1990 , OK, but it was marked with (V) in jira, so that was documented :)
Modified

Reply:
yes, my bad, should have edited Jira item. Thanks